test/from.js

```diff
@@ -27,7 +27,7 @@ export default {
 
         let usesThis = false;
 
-        Observable.from.call(_=> usesThis = true, []);
+        Observable.from.call(function(_) { usesThis = true; }, []);
```

Comment:
since the function isn't using `this`, there's no need for it to be a non-arrow function.

Reply:
Observable.from will attempt to use a callable `this` as a constructor. That's not legal for an arrow function:
```
$ node
> let usesThis = false;
undefined
> let f = _ => usesThis = true;
undefined
> new f();
TypeError: f is not a constructor
    at repl:1:1
    at ContextifyScript.Script.runInThisContext (vm.js:44:33)
    at REPLServer.defaultEval (repl.js:239:29)
    at bound (domain.js:301:14)
    at REPLServer.runBound [as eval] (domain.js:314:12)
    at REPLServer.onLine (repl.js:440:10)
    at emitOne (events.js:120:20)
    at REPLServer.emit (events.js:210:7)
    at REPLServer.Interface._onLine (readline.js:277:10)
    at REPLServer.Interface._line (readline.js:624:8)
```

Reply:
ahhh ok that makes sense. should this test then also ensure the following: 
```js
function usesThis() { usesThis = true; }
const result = Observable.from.call(usesThis, []);
assert(result instanceof usesThis);
```

Reply:
Asserting instanceof seems clearer to me, though it is perhaps slightly stricter as it disallows wrapping the result in the Observable under test. I'm not sure what the exact intention behind the test is.

Asserting usesThis would ensure the constructor was actually called (no Object.create).
Asserting instanceof ensures the result from using the custom constructor was not further wrapped.

```js
let usesThis = false;
function custom(_) {usesThis = true; }
let result = Observable.from.call(custom, []);

test._("Observable.from will use the 'this' value if it is callable")
.equals(usesThis, true)
.assert(result instanceof custom);
```